Cargo.toml

```diff
@@ -134,8 +134,9 @@ sqlite = ["nu-command/sqlite"]
 
 [profile.release]
 opt-level = "s"     # Optimize for size
-strip = "debuginfo"
+#strip = "debuginfo"
 lto = "thin"
+debug = true
```

Comment:
Let's not land this part... We have added `--profile profiling` exactly for this purpose

Reply:
fixed

Reply:
Thanks!